### PR TITLE
Add constructor options and equals method to FresnelParameters + tests.

### PR DESCRIPTION
**Pull request by brianzinn**
https://forum.babylonjs.com/t/fresnelparameters-ctor/9108/4

### CODE REVIEW

**Comment by brianzinn** [comment]
I didn't think there was a What's New worth mentioning :)

File: src/Materials/fresnelParameters.ts [hunk 1]
```diff
@@ -28,22 +89,32 @@ export class FresnelParameters {
...
-    public bias = 0;
+    public bias: number;
 
     /**
      * Defined the power exponent applied to fresnel term
      */
-    public power = 1;
+    public power: number;
+
+    public constructor(options: IFresnelParametersCreationOptions = {}) {
```
Comment (sebavan) [inline src/Materials/fresnelParameters.ts]: You need to add a comment on the constructor as we do in the other classes for the build to be ok :-)

File: src/Materials/fresnelParameters.ts [hunk 2]
```diff
@@ -57,37 +128,47 @@ export class FresnelParameters {
         return newFresnelParameters;
     }
 
+    /**
+     * Determines equality between FresnelParameters objects
+     * @param otherFresnelParameters defines the second operand
+     * @returns true if the power, bias, leftColor, rightColor and isEnabled values are equal to the given ones
+     */
+    public equals(otherFresnelParameters: DeepImmutable<FresnelParameters>): boolean {
```
Comment (sebavan) [inline src/Materials/fresnelParameters.ts]: If it is only for tests, should we introduce it here ?

Comment (brianzinn) [reply src/Materials/fresnelParameters.ts]: I added `equals(...)` as I use it for diffing out changes when I reconcile changes and need to update the BabylonJS scene.  If you think they are not useful then I can remove - I am okay to check all properties individually when I diff.

**Pull request merged**